scripts/dev.js

```diff
@@ -10,8 +10,9 @@ tasks.copyAssets('dev');
 
 console.log('[Webpack Dev]');
 console.log('-'.repeat(80));
-console.log('If you\'re developing Inject page,');
+console.log('In order to allow access to inject.js');
 console.log('please allow `https://localhost:3000` connections in Google Chrome,');
+console.log('which can be done by enabling #allow-insecure-localhost at chrome://flags');
```

Comment:
Sure, we can go with this new console message. We can always change it in the future if we get a different understanding.

As far as whether HTTPS is required or not - I'm pretty sure it is, but I can't remember why. It's generally speaking a good practice for all dev environment work to be in HTTPS anyway, since everything is moving to that.